fastlane/lib/fastlane/tools.rb

```diff
@@ -0,0 +1,20 @@
+module Fastlane
```

Comment:
That's a good point - probably not? We also need to consider this in https://github.com/fastlane/fastlane/pull/4473